src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (evento) => {
```

Comment:
The parameter name 'evento' is non-standard. Consider using 'event' instead, as it is the conventional name for event objects in JavaScript.